src/OrchardCore/OrchardCore.DisplayManagement/ModelBinding/PageModelUpdater.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace OrchardCore.DisplayManagement.ModelBinding
+{
+    public class PageModelUpdater : PageModel, IUpdateModel
+    {
+        private readonly PageModel _page;
+
+        public PageModelUpdater(PageModel page)
+        {
+            _page = page;
+            PageContext = page.PageContext;
+        }
+
+        public new ModelStateDictionary ModelState => _page.ModelState;
```

Comment:
We don't need this line as the base class retrieve the same through the PageContext